### ISSUE TITLE
Selections should be read by screenreader both in client and mashup. 

### ISSUE THREAD

**Issue by mlb76** [qlik-oss/sn-table#167]
Selections should be read by screenreader both in client and mashup.  At his time the number of selections made are read in the mashup, but not in the client. Maybe it would be good to read upp what is selected as well.

**Comment by haxxmaxx** [comment]
Closing this since we added a new issue from A360
